.gitignore

```diff
@@ -5,3 +5,6 @@
 
 #Artifacts from vscode
 .vscode/*
+
+#Rviz config
+geometric_controller/launch/config_file.rviz
```

Comment:
Why would you want to ignore the rviz config?

Reply:
Because i am always configuring rviz. So need always to commit. I can remove this condition on .gitignore if you like.

Reply:
Please remove it from the gitignore. You just need to not commit the rviz changes you did when you commit